R/assertions.R

```diff
@@ -1001,20 +959,24 @@ assert_function <- function(arg, params = NULL, optional = FALSE) {
 
 #' Assert Argument is a Parameter of a Function
 #'
+#' @description
+#' `r lifecycle::badge("deprecated")`
+#'
+#' This function is *deprecated*, please use `assert_function()` instead.
+#'
 #' @param arg The name of a function passed as a string
 #' @param params A character vector of function parameters
 #'
-#' @keywords assertion
-#' @family assertion
+#' @keywords deprecated
+#' @family deprecated
 #'
 #' @return
 #' The function throws an error if any elements of `params` is not an argument of
 #' the function given by `arg`
 #'
 #' @export
 assert_function_param <- function(arg, params) {
-  deprecate_warn("0.5.0", "assert_function_param()", "assert_function()", always = TRUE)
-  assert_function(arg = arg, params = params)
+  deprecate_stop("0.5.0", "assert_function_param()", "assert_function()", always = TRUE)
```

Comment:
Correct function was deprecated, but the roxygen documentation portion of it was misplaced